src/components/general/ExternalLink.css

```diff
@@ -0,0 +1,40 @@
+.external-container {
```

Comment:
No copyright in these files?

Reply:
The majority of the files here don't have any copyright header.
Should probably get fixed...